pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/json/users/requests/UserCreationPayload.java

```diff
@@ -22,7 +22,6 @@ public class UserCreationPayload {
     @SerializedName("email")
     private String email;
 
-    @NotEmpty
```

Comment:
Given this change, I think we want someting in `UserCreationRoute` around L52 so that we throw an error when this isn't present.  Am I right that it's no longer required just for OS1 user creation route?